{"name": "test_login_failed[data2]", "status": "passed", "attachments": [{"name": "log", "source": "9a5b3e06-eb97-422a-a7c7-3f62dc35b77f-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'name': '登录-反向用例-密码为空', 'username': 'caiyou', 'password': '', 'error_msg': '账号登录'}"}], "start": 1614175532583, "stop": 1614175536213, "uuid": "e40b9861-2419-4acc-82cb-0f2017ce480d", "historyId": "daee70692129a544e4ece7a1edb1fbca", "testCaseId": "130e6dce60bc4ef9d882c5f0465dc1d6", "fullName": "cases.test_login.TestLogin#test_login_failed", "labels": [{"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "ruslan"}, {"name": "thread", "value": "3064-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_login"}]}